language/English/strings.po

```diff
@@ -12792,7 +12792,12 @@ msgctxt "#25009"
 msgid "The menu of this Blu-ray is not supported"
 msgstr ""
 
-#empty strings from id 25010 to 29799
+#: xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#25010"
```

Comment:
string 21396 is used by many skins, so not a good idea to change that one imo.

> Are I** skins still compatible with Helix?
not 100% no. it's not that things are broken completely, far from that.
but there are some minor changes, mostly in the pvr area.